example.js

```diff
@@ -19,6 +21,13 @@ class CheckFeatures {
   }
 
   process(tweet) {
+    if (tweet.place) {
+      var hashtagList = tweet.entities.hashtags
+      for (var i = 0; i < hashtagList.length; i++) {
+        console.log('hashtag: ' + hashtagList[i]);
+        db.insertHashtagStats(tweet.place.country, hashtagList[i].text);
+      }
+    }
```

Comment:
You should create your own processor object if you're adding new stuff, instead of modifying `CheckFeatures`. I was intending `CheckFeatures` to get some stats on what information is included in tweets.